### PR TITLE
Adding section how to work with OCI helm charts

### PR DESCRIPTION
**Pull request by OrlinVasilev**

### CODE REVIEW

**Comment by OrlinVasilev** [comment]
@Vad1mo @wy65701436 can you please check if it makes sense to have to that here will add some screen shots from the UI as well

**Review by Vad1mo — CHANGES_REQUESTED** [review]
This is valuable. 
I have only one question, why is the same content twice but in different locations?

**Comment by OrlinVasilev** [comment]
@Vad1mo  it's a mistake of course :) started at one place and moved it to another without actually fixing it :) thanks!

**Review by Vad1mo — APPROVED** [review]
very nice

File: docs/working-with-projects/working-with-oci/working-with-helm-oci-charts.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+You can now pull a chart:
+
+```sh
+helm pull oci://<harbor_address>/<project>/<chart_name> --version <version>
+```
+
+e.g.
+```sh
+helm pull oci://demo.goharbor.io/oci/demo --version 0.1.0
```
Comment (zyyw) [inline docs/working-with-projects/working-with-oci/working-with-helm-oci-charts.md:26]: maybe change `oci://demo.goharbor.io/oci/demo` to `oci://demo.goharbor.io/oci/oci-charts` to make it consistent between the `docs/img/oci/oci-chart-main-view.png` image and the text.

File: docs/working-with-projects/working-with-oci/working-with-helm-oci-charts.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Push the OCI Helm Chart:
+
+```sh
+helm push <chart_name_and_version>.tgz oci://<harbor_address>/<project>
+```
+
+e.g.
+```sh
+helm push example-0.1.0.tgz oci://demo.goharbor.io/oci
```
Comment (zyyw) [inline docs/working-with-projects/working-with-oci/working-with-helm-oci-charts.md:55]: maybe change `helm push example-0.1.0.tgz oci://demo.goharbor.io/oci` to `helm push oci-chart-example-0.1.0.tgz oci://demo.goharbor.io/oci-charts` to make it consistent between the `docs/img/oci/oci-chart-main-view.png` image and the text.

**Review by zyyw — APPROVED** [review]
LGTM, while left some comments

**Pull request merged**